_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.html

```diff
@@ -39,13 +51,26 @@
 
 One model often referenced is the so-called, and poorly named, “Spotify model”, which is only used here as a demonstration of an alternate model whereby product owners and managers needn't be the same person. While this is generally the model that was used, it is abstracting away complexities that make it difficult for this model to work well in a government context. For example, the director at the top who is responsible for the management of the product is not the director for all of the teams in the box, though he is responsible for the platform, which makes reporting, hierarchies, and performance management a potential issue.
 
-![Spotify Model]({{ site.baseurl }}/assets/images/TTBlog-Figure3.PNG)
-_Figure 3_
+![Spotify Model]({{ site.baseurl }}/assets/images/TTBlog-Figure3New.PNG)
```

Comment:
Just a quick question: Why is the filename changing? I don't see a new image either in master nor in this branch.

Thanks!

Reply:
I didn't want to overwrite the original file so I changed the name - the new version of the image does not include the word 'tribe'. Its odd you don't see it though...its not in the master branch but I see it in assets/images folder in the ttBlog_images_fix branch.  Maybe I didn't do the commit correctly?

Reply:
Aye! Nope, the image actually is in this branch but the extension is `.png` in the filename whereas the code in the MD file is calling for `.PNG`. Sorry about the confusion! :D